app/assets/javascripts/3dbio_viewer/src/webapp/components/network/UniProtAccessionTextArea.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./Network.css";
+
+interface UniProtAccessionTextAreaProps {
+    textareaValue: string;
+    onTextareaChange: (e: string) => void;
+}
```

Comment:
As the component refers to a text area, its props can be simply `value` and `onChange`.